test/ffmpeg-vaapi/transcode/transcoder.py

```diff
@@ -0,0 +1,83 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+import os
+
+@slash.requires(have_ffmpeg)
+@slash.requires(have_ffmpeg_vaapi_accel)
+class TranscoderTest(slash.Test):
+  def before(self):
+    self.refctx = []
+
+  def transcode(self):
+    if vars(self).get("ttype", None) == 'hwhw':
+      self.mformat = transcodeformat_hwhw_swhw(self.dstformat)
```

Comment:
Generally, `format` implies "pixel format" in most places we use this name.  So this seems to confuse me when used in this context.  Can we call it `mcodec` or something, instead?

Reply:
changed mformat to mcodec